### ISSUE TITLE
Invalid code crashes enu on open

### ISSUE THREAD

**Issue by juancarlospaco** [dsrw/enu#9]
On Edit mode, type something invalid on a block like a typo or something:

```nim
import grid
forward 3
INVALID
```

- Press <kbd> ESC </kbd>
- Enu closes.
- Open Enu again.
- Enu crashes.

```console
$ ./enu
error: undeclared identifier: 'invalid'
/home/juan/.local/share/enu/default/scripts/grid_14.nim done.
undeclared identifier: 'invalid'
segfaults.nim(76)        segfaultHandler
Error: unhandled exception: Could not access value because it is nil. [NilAccessDefect]

$
```

**Comment by juancarlospaco** [comment]
I think this should be the high priority compared to the other bugs, is frequent and annoying,
you have to dig deep on the hidden folders to delete or edit the Nim file with the typo. :neutral_face:

**Comment by dsrw** [comment]
https://github.com/dsrw/enu/releases/tag/v0.1.1 has been published, which should fix this issue.

**Comment by juancarlospaco** [comment]
Awesome!, downloading... :smile_cat:

**Comment by dsrw** [comment]
v0.1.1 had a showstopper regression that made it impossible to drop blocks on the ground. I pushed it out without much testing unfortunately.

I removed v0.1.1 from the releases page and published v0.1.2 with a fix.
